### PR TITLE
Few commands added

### PR DESCRIPTION
**Pull request by dotsbeauty**
### Checklist:
- [x] Read our [contributing guidelines](../docs/CONTRIBUTING.md).
- [x] Search for duplicates.
- [x] Used an informative name for this pull request.

### Follow-up
- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: CheatSheets/html-cheatsheet.md [hunk 1]
```diff
@@ -144,3 +151,19 @@
...
+
+**[🔼Back to Top](#table-of-contents)**
+
+## Graphics
+
+| Command                    | Description                                      |
+| `<video> … </video>`       | Used to show a video on a web page.              |
+| `<audio> … </audio>`       | Used to play an audio file on a web page.        |
+
+**[🔼Back to Top](#table-of-contents)**
```
Comment (crescentpartha) [inline CheatSheets/html-cheatsheet.md:169]: Those lines of code `break the table format`.
- Double `Graphics header`
- Double `canvas` added

File: CheatSheets/html-cheatsheet.md [hunk 2]
```diff
@@ -71,6 +74,10 @@
 | `<section> ... </section>`       | Defines a section in a document                                                             |
 | `<summary> ... </summary>`       | Defines a visible heading for a `<details>` element                                         |
 | `<time> ... </time>`             | Defines a date/time                                                                         |
+| `<pre> ... </pre>`               | Preserve both spaces and linebreaks                                                         |
+| `<code> ... </code>`             | Defines sample output from a computer program                                               |
```
Comment (crescentpartha) [inline CheatSheets/html-cheatsheet.md:78]: Already exist.

**Review by crescentpartha — CHANGES_REQUESTED** [review]
Please, modify based on the requirements.

**Review by dotsbeauty — COMMENTED** [review]
Conflict resolved

File: CheatSheets/html-cheatsheet.md [hunk 3]
```diff
@@ -145,3 +152,19 @@
...
+
+**[🔼Back to Top](#table-of-contents)**
+
+## Media
+
+| Command                    | Description                                      |
+| `<video> … </video>`       | Used to show a video on a web page.              |
+| `<audio> … </audio>`       | Used to play an audio file on a web page.        |
+
+**[🔼Back to Top](#table-of-contents)**
```
Comment (crescentpartha) [inline CheatSheets/html-cheatsheet.md:170]:
```suggestion
## Graphics

| Command                    | Description                                                        |
| --------------------- | ------------------- |
| `<canvas> … </canvas>`     | Used to draw graphics on a web page using javascript.    |
| `<svg> … </svg>`           | Used to defines vector-based graphics in XML format.               |

**[🔼Back to Top](#table-of-contents)**

## Media

| Command                    | Description                                      |
| --------------------- | ------------------- |
| `<video> … </video>`       | Used to show a video on a web page.              |
| `<audio> … </audio>`       | Used to play an audio file on a web page.        |

**[🔼Back to Top](#table-of-contents)**
```
The table is broken again. [here](https://github.com/dotsbeauty/CheatSheets-for-Developers/blob/main/CheatSheets/html-cheatsheet.md#graphics)

**Review by crescentpartha — CHANGES_REQUESTED** [review]
Please, modify the broken tables.

**Review by crescentpartha — APPROVED** [review]
Approved.

**Pull request merged**